base/terminfo_data.jl

```diff
@@ -67,6 +67,7 @@ for line in eachline(seekstart(standard_caps))
         @warn "Unrecognised capability type: $type"
         continue
     end
+    description = replace(description, r"\\f[BPRI]?" => "")
```

Comment:
Why is this necessary? Would it make sense to add a one-line comment explaining it?

Reply:
It's been long enough that I have a hard time remembering, but IIRC this strips out some (unwanted) formatting directives from the text that were introduced in the update.